Userland/Services/WebServer/main.cpp

```diff
@@ -51,6 +51,11 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
+    if (port < 1024 && geteuid() != 0) {
+        warnln("Listening on port {} requires root privileges", port);
+        return 1;
+    }
```

Comment:
We should probably not duplicate that check here into userland, it may end up being inaccurate already or getting inaccurate some time in the future. I assume it currently just prints a "Permission denied" error or something similar?